src/databricks/labs/ucx/source_code/notebook.py

```diff
@@ -0,0 +1,331 @@
+from __future__ import annotations  # for type hints
```

Comment:
do we really need it?..

Reply:
yes for forward type hints

Reply:
without it, you have to use `'MyType'` instead of `MyType`, and `'MyType' or None` instead of `MyType | None`